addons/web/static/src/core/orm_service.js

```diff
@@ -120,105 +109,86 @@ export class ORM {
         return this.rpc(url, params, { silent: this._silent });
     }
 
-    create(model, state, ctx) {
+    create(model, state, context) {
         validateObject("state", state);
-        return this.call(model, "create", [state], { context: ctx });
+        return this.call(model, "create", [state], { context });
```

Comment:
While we are at it, could we allow create to work in batch instead of enforcing a single element and hardcoding the array?

Reply:
done